src/com/timezynk/mongo/util.clj

```diff
@@ -22,12 +22,21 @@
   (reset! mongo-uri {:uri uri
                      :options options}))
 
-(def ^:no-doc make-connection
-  (delay (let [uri (or @mongo-uri
-                       {:uri (System/getenv "MONGO_URI")}
-                       (throw (MongoClientException. "Call set-mongo-uri! or define MONGO_URI environment variable prior to calling wrap-mongo")))]
-           (log/info "MongoDB servers" uri)
-           (connection-method (:uri uri) (:options uri)))))
+(defn current-uri []
+  @mongo-uri)
```

Comment:
`current-uri` isn't used.

Reply:
I need this in `tzbackend`. I target it in there with `with-redefs` to build fixtures.

Reply:
It turns out that `current-uri` was not necessary, after all.

It is better to remove it from this PR to keep the scope narrow.